src/store/middleware/model-poller.ts

```diff
@@ -335,13 +356,15 @@ export const modelPollerMiddleware: Middleware<
       if (!conn) {
         return;
       }
-      let crossModelQueryResponse: CrossModelQueryFullResponse;
+      let crossModelQueryResponse: CrossModelQueryResponse | Error;
       try {
         crossModelQueryResponse = await crossModelQuery(conn, query);
       } catch (error) {
         console.error("Could not perform cross model query:", error);
         crossModelQueryResponse =
```

Comment:
Instead of passing this to `updateCrossModelQuery` we should instead add a `storeCrossModelQueryError` action/reducer and dispatch that here.